guides/common/modules/proc_using-ansible-job-templates-for-package-and-errata-actions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Using_Ansible_Job_Templates_for_Package_and_Errata_Actions{context}"]
+= Using Ansible job templates for package and errata actions
+
+By default, {Project} is configured to use the Script provider templates for remote execution jobs.
+If you prefer using the Ansible provider templates for your remote jobs, you can configure {Project} to use the Ansible provider templates for remote execution features that take a search query input.
```

Comment:
With my pedantic hat on:
This reads as if this applies to "remote execution features that take a search query input", but features don't really take inputs, job templates do.

Reply:
@jeremylenz Thoughts on how this should be written?